index.js

```diff
@@ -75,8 +82,41 @@ module.exports = function (source, map) {
     }
   });
 
-  function relativePath(fullPath) {
-    return path.relative(rootPath, fullPath);
+  function getAliases (options) {
+    // TODO: Investigate if there is a way to leverage
+    // the Webpack API to get all the aliases
+    var resolveAliases = options.resolve && options.resolve.alias;
+    var resolveLoaderAliases = options.resolveLoader && options.resolveLoader.alias;
+    return Object.assign({}, resolveAliases || {}, resolveLoaderAliases || {});
```

Comment:
Using a shim for now. I'll look into to the testing bit next. I'll try putting out a PR for the ES6 (& `npm prepublish`) stuff after that unless you find the time to do that first :)